controllers/fenceagentsremediation_controller_test.go

```diff
@@ -69,39 +71,54 @@ var _ = Describe("FAR Controller", func() {
 	underTestFAR = newFenceAgentsRemediation(validNodeName, fenceAgentIPMI, testShareParam, testNodeParam)
 
 	Context("Functionality", func() {
-		Context("buildFenceAgentParams", func() {
-			When("FAR's name isn't a node name", func() {
+		Context("buildFenceAgentParams - check CR name", func() {
+			When("FAR's name doesn't match a node name", func() {
 				It("should fail", func() {
 					underTestFAR.ObjectMeta.Name = dummyNodeName
 					_, err := buildFenceAgentParams(underTestFAR)
 					Expect(err).To(HaveOccurred())
+					Expect(err).To(Equal(errors.New(errorBuildingFAParams)))
 				})
+			})
+			When("FAR's name does match a node name", func() {
 				It("should succeed", func() {
 					underTestFAR.ObjectMeta.Name = validNodeName
 					_, err := buildFenceAgentParams(underTestFAR)
```

Comment:
just fyi, this is a bit shorter and nicer:
`Expect(buildFenceAgentParams(underTestFAR)).To(Succeed())`

Reply:
I tried that, and it failed since on success the function returns a string list.

Error :

>    [FAILED] Expected an error-type.  Got:
      <[]string | len:6, cap:8>: [
          "--username=admin",
          ...